python/ray/services.py

```diff
@@ -34,13 +35,14 @@
 # important because it determines the order in which these processes will be
 # terminated when Ray exits, and certain orders will cause errors to be logged
 # to the screen.
-all_processes = OrderedDict([(PROCESS_TYPE_WORKER, []),
+all_processes = OrderedDict([(PROCESS_TYPE_MONITOR, []),
```

Comment:
When I run `./scripts/start_ray.sh --head` on this branch, the script raises an exception when it exits and prints

```
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/opt/conda/lib/python2.7/atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
  File "/home/ubuntu/ray/python/ray/worker.py", line 963, in cleanup
    assert(len(processes) == 0)
AssertionError
Error in sys.exitfunc:
Traceback (most recent call last):
  File "/opt/conda/lib/python2.7/atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
  File "/home/ubuntu/ray/python/ray/worker.py", line 963, in cleanup
    assert(len(processes) == 0)
AssertionError
```